packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -0,0 +1,125 @@
+import autoscaling = require('@aws-cdk/aws-autoscaling');
+import ec2 = require('@aws-cdk/aws-ec2');
+import cdk = require('@aws-cdk/cdk');
+import { cloudformation, ClusterArn} from './ecs.generated';
+
+export interface ClusterProps {
+    /**
+     * A name for the cluster.
+     *
+     * @default CloudFormation-generated name
+     */
+    clusterName?: string;
+
+    /**
+     * The VPC where your ECS instances will be running
+     */
+    vpc: ec2.VpcNetworkRef;
+
+    /**
+     * Whether or not the containers can access the instance role
+     *
+     * @default false
+     */
+    containersAccessInstanceRole?: boolean;
```

Comment:
How about we define this for each `ContainerDefinition` individually, and automatically aggregate over all services assigned to this cluster? Then it's one less thing the user has to worry about getting right.

Since we also want to support a non-complete view of the world (that the services assigned to this cluster are not defined in CDK, but added at some "later point"), I suggest we take this default behavior if the user doesn't supply the value, and otherwise (if they supply `true` or `false` just take them at their word).